### PR TITLE
PMM-949: fix mapping MySQL bool variables to golang bool

### PR DESCRIPTION
**Pull request by arvenil**
* PMM-949:
  ```diff --git a/pct/sys.go b/pct/sys.go
  index 75d280e..b022bb8 100644
  --- a/pct/sys.go
  +++ b/pct/sys.go
  @@ -194,7 +194,7 @@ func AtLeastVersion(v1, v2 string) (bool, error) {
   func ToBool(s string) bool {
          s = strings.ToLower(s)
          switch s {
  -       case "yes", "true", "on":
  +       case "yes", "true", "on", "1":
                  return true
          default:
                  return false
  ```
* Typo
  ```diff --git a/qan/analyzer/mysql/mysql.go b/qan/analyzer/mysql/mysql.go
  index b71b3a9..dc0f4c6 100644
  --- a/qan/analyzer/mysql/mysql.go
  +++ b/qan/analyzer/mysql/mysql.go
  @@ -197,18 +197,18 @@ func (m *MySQLAnalyzer) GetDefaults(uuid string) map[string]interface{} {
          mysqlConn := m.mysqlConnFactory.Make(mysqlInstance.DSN)
          config.ReadMySQLConfig(mysqlConn) // Read current values
          return map[string]interface{}{
                  "CollectFrom":            collectFrom,
                  "Interval":               interval,
                  "LongQueryTime":          config.DEFAULT_LONG_QUERY_TIME,
                  "MaxSlowLogSize":         config.DEFAULT_MAX_SLOW_LOG_SIZE,
                  "RemoveOldSlowLogs":      config.DEFAULT_REMOVE_OLD_SLOW_LOGS,
                  "ExampleQueries":         exampleQueries,
                  "SlowLogVerbosity":       config.DEFAULT_SLOW_LOG_VERBOSITY,
                  "RateLimit":              config.DEFAULT_RATE_LIMIT,
                  "LogSlowAdminStatements": config.DEFAULT_LOG_SLOW_ADMIN_STATEMENTS,
  -               "LogSlowSlaveStatemtents":config.DEFAULT_LOG_SLOW_SLAVE_STATEMENTS,
  +               "LogSlowSlaveStatements": config.DEFAULT_LOG_SLOW_SLAVE_STATEMENTS,
                  "WorkerRunTime":          config.DEFAULT_WORKER_RUNTIME,
                  "ReportLimit":            config.DEFAULT_REPORT_LIMIT,
          }
   }
  ```
* TravisCI: Testing on multiple MySQL and MongoDB versions (travis matrix) with MySQL and MongoDB provided by docker. 🍻 to @AlekSi for cool usage of `docker-compose`. However we should probably switch at some point to https://github.com/ory/dockertest
  build__339_-_percona_qan-agent_-_travis_ci
* Add slack notifications
* Some cleanups

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,45 @@
+sudo: required
+
 language: go
 
 go:
   - 1.7.x
   - 1.8.x
```
Comment (AlekSi) [inline .travis.yml]: I would add `master` – 1.9 is just around the corner with changes in `database/sql`.

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,45 @@
+sudo: required
```
Comment (AlekSi) [inline .travis.yml:2]: I would also add `dist: trusty`.

File: .travis.yml [hunk 3]
```diff
@@ -1,29 +1,45 @@
...
+matrix:
+  exclude:
+    - go: 1.7.x
+  include:
+    - go: 1.7.x
+      env: MYSQL_IMAGE=mysql:5.7 MONGODB_IMAGE=mongo:3.5
+
 services:
-  - mongodb
   - mysql
```
Comment (AlekSi) [inline .travis.yml]: Do you need host's MySQL?

**Review by AlekSi — COMMENTED** [review]
Feel free to steal something from proxysql_exporter: docker-compose.yml, .travis.yml, CONTRIBUTING.md.

File: .travis.yml (hunk 3 above)
Comment (arvenil) [reply .travis.yml]: no, but I need mysql tools e.g. `mysql` cmd, they come with the package

File: .travis.yml (hunk 2 above)
Comment (arvenil) [reply .travis.yml:2]: Is there a point to do this? Switches automatically to trusty with `sudo: required`.

File: .travis.yml (hunk 1 above)
Comment (arvenil) [reply .travis.yml]: Yeah, no worries about this will do in future. Right now I'm trying to minimize number of builds ;)

File: .travis.yml (hunk 2 above)
Comment (AlekSi) [reply .travis.yml:2]: It switches to some distro where sudo is provided, it just happened to be trusty at the moment. If you want trusty (and you do for that apt line), it's better to specify it.

Comment (arvenil) [reply .travis.yml:2]: I don't want that apt line, that's the issue :D I theory I could use something like `$(lsb_release -s -c)` but I'm not sure if config file allows something like:

```yaml
sources:
  - sourceline: "deb http://repo.percona.com/apt $(lsb_release -s -c) testing"
```

Comment (arvenil) [reply .travis.yml:2]: It works 🎉 !

It gets converted to:
```sh
echo "deb http://repo.percona.com/apt $(lsb_release -s -c) testing" | sudo tee -a /etc/apt/sources.list > /dev/null
```

**Review by askomorokhov — APPROVED** [review]
LGTM

**Pull request merged**